.taskcluster.yml

```diff
@@ -5,385 +5,278 @@
-                          TASKGRAPH_HEAD_REPOSITORY: https://hg.mozilla.org/ci/${taskgraph.branch}
-                          TASKGRAPH_HEAD_REV: ${taskgraph.revision}
-                          TASKGRAPH_REPOSITORY_TYPE: hg
-                          REPOSITORIES: {$json: {appservices: "Application Services", taskgraph: "Taskgraph"}}
-                          HG_STORE_PATH: /builds/worker/checkouts/hg-store
-                          ANDROID_SDK_ROOT: /builds/worker/android-sdk
-                          MOZ_FETCHES_DIR: /builds/worker/checkouts/src
-                        - $if: 'tasks_for in ["github-pull-request"]'
-                          then:
-                            APPSERVICES_PULL_REQUEST_TITLE: '${event.pull_request.title}'
-                            APPSERVICES_PULL_REQUEST_NUMBER: '${event.pull_request.number}'
-                    features:
-                      taskclusterProxy: true
-                      chainOfTrust: true
-                    # Note: This task is built server side without the context or tooling that
-                    # exist in tree so we must hard code the hash
-                    image:
-                      mozillareleases/taskgraph:decision-mobile-6020473b1a928d8df50e234a7ca2e81ade2220a4fb5fbe16b02477dd64a49728@sha256:98d226736b7d03907114bf37938002b90e8a37cbe3a297690e349f1ddddb1d7c
+                                                Action triggered by clientID `${clientId}`
+                                        else:
+                                            name: "Decision Task for cron job ${cron.job_name}"
+                                            description: 'Created by a [cron task](https://firefox-ci-tc.services.mozilla.com/tasks/${cron.task_id})'
+                          provisionerId: "app-services-${level}"
+                          workerType: "decision"
+                          tags:
+                              $if: 'tasks_for in ["github-push", "github-pull-request"]'
+                              then:
+                                  kind: decision-task
+                              else:
+                                  $if: 'tasks_for == "action"'
+                                  then:
+                                      kind: 'action-callback'
+                                  else:
+                                      $if: 'tasks_for == "cron"'
+                                      then:
+                                          kind: cron-task
+                          routes:
+                            $flattenDeep:
+                              - checks
+                              - $if: 'level == "3"'
+                                then:
+                                    # TODO Bug 1601928: Make this scope fork-friendly once ${project} is better defined. This will enable
+                                    # staging release promotion on forks.
+                                    - $if: 'tasks_for == "github-push"'
```

Comment:
I really need to fix these routes at some point too, but will address as a follow-up. For now, let's keep things as they are (custom app-services indexes). We'll switch over to standard ones as part of https://bugzilla.mozilla.org/show_bug.cgi?id=1631839